lab-nicole/lib/server-toggle.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const debug = require('debug')('mp3uploader:server-toggle');
+const mongoose = require('mongoose');
+
+module.exports = exports = {};
+
+exports.serverOn = function(server, done) {
```

Comment:
I would encourage you to look into other ways to create a server toggle that would avoid potential duplicate server issues like we have had in the past. Think about promises, and think about where you actually start your server.